products/opensuse/main.pm

```diff
@@ -939,6 +939,11 @@ elsif (get_var('INSTALL_LTP')) {
     loadtest 'kernel/boot_ltp';
     loadtest 'kernel/shutdown_ltp';
 }
+elsif (get_var('LTP_SETUP_NETWORKING')) {
```

Comment:
I suppose you are running this after install_ltp then publishing a disk image. However it probably only takes a few seconds to do the actual setup so we could just run ltp_setup_networking before each test. The same way I am running create_junkfile_ltp. Publishing disk images is a lot of work for the system.

We could also run it as part of install_ltp if you think there is no chance of it ever interfering with the normal tests. Probably not worth the risk though.